src/main/java/org/greencity/service/HttpService.java

```diff
@@ -10,113 +10,122 @@
 import org.apache.http.HttpResponse;
 import org.apache.http.HttpStatus;
 import org.apache.http.StatusLine;
+import org.apache.http.client.ClientProtocolException;
 import org.apache.http.client.methods.HttpPost;
 import org.apache.http.conn.HttpHostConnectException;
 import org.apache.http.entity.ContentType;
 import org.apache.http.entity.StringEntity;
-import org.apache.http.impl.client.HttpClients;
+import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.util.EntityUtils;
 import org.greencity.constant.EnvVar;
 import org.greencity.constant.LogMessage;
 import org.greencity.constant.LogSource;
 import org.greencity.dto.LogsRequestDto;
 import org.greencity.dto.LogsResponseDto;
 import org.greencity.entity.LokiChunk;
+import org.greencity.exceptions.HttpRequestExecutionException;
+import org.greencity.exceptions.InvalidProtocolException;
+import org.greencity.exceptions.UnableToConnectException;
+import org.greencity.exceptions.UnexpectedResponseException;
 import org.greencity.util.LokiAgentLogger;
-
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.logging.Logger;
 
-public class HttpService {
+import static org.greencity.constant.LogMessage.UNABLE_TO_EXECUTE_HTTP_REQUEST;
 
-    private static final Logger log = LokiAgentLogger.getLogger(HttpService.class);
+public class HttpService {
```

Comment:
can you please add a space in between here
```suggestion
public class HttpService {

```